spec/factories/content_item.rb

```diff
@@ -9,6 +9,7 @@
     update_type 'minor'
     routes { [{ 'path' => base_path, 'type' => 'exact' }] }
     phase nil
+    analytics_identifier nil
```

Comment:
Do we need to add this to the factories?  Won't it default to `nil` anyway?  (I don't think it was needed for phase either...)